rpc/src/v1/helpers/dispatch/mod.rs

```diff
@@ -0,0 +1,381 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity Ethereum.
+
+// Parity Ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Utilities and helpers for transaction dispatch.
+
+pub(crate) mod light;
+mod full;
+mod prospective_signer;
+
```

Comment:
This is a good change i.e., to break-up `dispatcher` into different mods however I haven't reviewed that code. I assume it is just moved?

Reply:
Yeah, mostly moved, but I had to do some small changes to support the new `Accounts` trait instead of using `AccountProvider` directly.

Reply:
all right, we should probably run some manual tests on the `light-client` before merging because rpc integration tests are still missing